matchzoo/generators/pair_generator.py

```diff
@@ -131,12 +132,13 @@ def _get_batch_of_transformed_samples(
         trans_index = []
         steps = self._num_neg + 1
         for item in index_array:
-            trans_index.extend(list(range(item*steps, (item+1)*steps)))
+            trans_index.extend(list(range(item * steps, (item + 1) * steps)))
         batch_x = {}
         batch_y = self._relation.iloc[trans_index, 2].values
 
-        columns = self._left.columns.values.tolist() + \
-            self._right.columns.values.tolist() + ['id_left', 'id_right']
+        left_vals = self._left.columns.values.tolist()
```

Comment:
change `left_vals` to `left_columns`, and `right_vals` to `right_columns`.

Reply:
sure